### PR TITLE
added thanks to all the contributors in readme.md

### LINKED ISSUE #47: Add Thanks to the all Contributors & Support in README.md
We should add Thanks to the all Contributors & Support in README like [LinkFree](https://github.com/EddieHubCommunity/LinkFree)

Thanks and Support

### PR DESCRIPTION
**Pull request by codematrixaditya**
- fixes #47 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/EventCalendar/pull/72"></a>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,3 +37,12 @@ Here is an example file...
 5. Run `npm run generate` to generate the calendar list data (do not change this file, it will be ignored by git)
 6. Run `npm start` to start the server
 7. Visit http://localhost:3000
+
+## 💪 Thanks to the all Contributors
+
+Thanks a lot for spending your time in helping LinkFree grow. Thanks a lot! Keep rocking 🍻
```
Comment (suitupstan) [inline README.md]:
```suggestion
Thanks a lot for spending your time helping EventCalender grow. Thanks a lot! Keep rocking 🍻
```

**Review by suitupstan — CHANGES_REQUESTED** [review]

**Review by suitupstan — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -37,3 +37,12 @@ Here is an example file...
...
 7. Visit http://localhost:3000
+
+## 💪 Thanks to the all Contributors
+
+Thanks a lot for spending your time helping EventCalender grow. Thanks a lot! Keep rocking 🍻
+[contributors](https://github.com/EddieHubCommunity/EventCalendar/graphs/contributors)
+
+## 🙏 Support
+
+This project needs a ⭐️ from you. Don't forget to leave a star ⭐️
```
Comment (suitupstan) [inline README.md]: Not sure if we need this line 🤔

**Review by suitupstan — CHANGES_REQUESTED** [review]

**Review by suitupstan — APPROVED** [review]

**Review by Vyvy-vi — APPROVED** [review]
lgtm!

**Review by shreyaansjain06 — APPROVED** [review]

**Comment by EmmaDawsonDev** [comment]
This change looks good but there is currently a merge conflict caused by a previous pull request being merged into the same file. If you can fix the merge conflict we can get this merged in.

**Review by suitupstan — COMMENTED** [review]
Please check the comments above.

File: README.md [hunk 3]
```diff
@@ -42,3 +42,8 @@ Here is an example file...
 ## 🛡️ License
 
 This project is licensed under the MIT License - see the [`LICENSE`](LICENSE) file for details.
+
+## 💪 Thanks to the all Contributors
+
+Thanks a lot for spending your time helping EventCalender grow. Thanks a lot! Keep rocking 🍻
```
Comment (suitupstan) [inline README.md]:
```suggestion
Thanks a lot for spending your time helping EventCalendar grow. Thanks a lot! Keep rocking 🍻

[![Contributors](https://contrib.rocks/image?repo=EddieHubCommunity/LinkFree)](https://github.com/EddieHubCommunity/LinkFree/graphs/contributors)
```

File: README.md [hunk 4]
```diff
@@ -42,3 +42,8 @@ Here is an example file...
...
+[contributors](https://github.com/EddieHubCommunity/EventCalendar/graphs/contributors)
```
Comment (suitupstan) [inline README.md]:
```suggestion
```

**Review by suitupstan — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -42,3 +42,7 @@ Here is an example file...
 ## 🛡️ License
 
 This project is licensed under the MIT License - see the [`LICENSE`](LICENSE) file for details.
+
+## 💪 Thanks to the all Contributors
+
+Thanks a lot for spending your time helping EventCalendar grow. Thanks a lot! Keep rocking 🍻
```
Comment (suitupstan) [inline README.md:48]:
```suggestion
Thanks a lot for spending your time helping EventCalendar grow. Thanks a lot! Keep rocking 🍻

[![Contributors](https://contrib.rocks/image?repo=EddieHubCommunity/EventCalendar)](https://github.com/EddieHubCommunity/LinkFree/graphs/contributors)
```

**Review by suitupstan — CHANGES_REQUESTED** [review]

**Pull request merged**